### PR TITLE
[CIGolang]change profile for 4.11

### PR DESCRIPTION
**Pull request by kuiwang02**
/cc @Xia-Zhao-rh 

1, change to qe cluster-profile.
2, take qe workflow if it exists (for few test, there is no workflow ready. still take chain.)
3, move periodics job to variant. it reduce the quota. if we keep it in the original, it will generate more build. For example,
if we add one test in master.yaml, it will be in release-4.11.yaml and release-4.12.yaml. it means it run 3times for same test. it will occupy more quota. after we move it to variant, there is only one test in variant.yaml. it only run 1 times for the test.

### CODE REVIEW

**Comment by kuiwang02** [comment]
will review step shell script again.

**Comment by kuiwang02** [comment]
1, change to qe cluster-profile.
2, take qe workflow if it exists (for few test, there is no workflow ready. still take chain.)
3, move periodics job to variant. it reduce the quota. if we keep it in the original, it will generate more build. For example,
if we add one test in master.yaml, it will be in release-4.11.yaml and release-4.12.yaml. it means it run 3times for same test. it will occupy more quota. after we move it to variant, there is only one test in variant.yaml. it only run 1 times for the test.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+      cpu: "2"
+      memory: 8Gi
+  e2e-aws:
+    requests:
+      cpu: "4"
+      memory: 4Gi
+test_binary_build_commands: make test
+tests:
+- as: extended-aws-ipi
+  cron: 0 2 */5 * *
```
Comment (jianzhangbjz) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml:73]: Every 5 days?

Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml:73]: I will change it to 3days

**Pull request merged**